downloader/src/main/kotlin/VcsHost.kt

```diff
@@ -338,7 +338,8 @@ enum class VcsHost(
             val unknownVcsInfo = VcsInfo.EMPTY.copy(url = projectUrl)
             val projectUri = projectUrl.toUri().getOrElse { return unknownVcsInfo }
 
-            fun URI.isTfsGitUrl() = path != null && host != null &&
+            fun URI.isTfsGitUrl() =
+                path != null && host != null &&
```

Comment:
This line and the line below would now actually fit in one line. And shouldn't they according to the rules?

Reply:
The rule does not touch existing line breaks within the function body, it only checks if the first line of the function body would fit on the same line as the signature. I have manually fixed it here.